cmd/serverless-init/cloudservice/appservice.go

```diff
@@ -35,8 +37,8 @@ func (a *AppService) GetTags() map[string]string {
 	tags := map[string]string{
 		"app_name":   appName,
 		"region":     region,
-		"origin":     a.GetOrigin(),
-		"_dd.origin": a.GetOrigin(),
+		"origin":     AppServiceOrigin,
+		"_dd.origin": AppServiceOrigin,
```

Comment:
This seemed like a redundant function call, but let me know if it's here for some special case.